src/plugin.json

```
@@ -4,10 +4,8 @@
   "id": "gel",
   "executable": "gel",
   "backend": true,
-
-  "version": "0.4.0",
-
   "info": {
+    "version": "0.4.0",
```

```suggestion
    "version": "0.5.0",
```

Will tag as 5 when merged - want keep version in file matching the git tag.

---

Thanks @kylebrandt